stable_baselines3/common/callbacks.py

```diff
@@ -342,9 +339,6 @@ def _log_success_callback(self, locals_: Dict[str, Any], globals_: Dict[str, Any
         :param globals_:
         """
         info = locals_["info"]
-        # VecEnv: unpack
```

Comment:
the code below should be updated too (counting successes)

Reply:
I think this code works as is. The evaluation code unpacks the results of the vector env so that the callback can count scalar values. For example the test on line 105 of test_callbacks.py passes.

Reply:
alright, I did not realize that you were unpacking later